src/modules/grubcfg/main.py

```diff
@@ -85,6 +86,30 @@ def get_zfs_root():
     return None
 
 
+def update_existing_config(default_grub, grub_config_items):
+    """
+    Updates the existing grub configuration file with any items present in @p grub_config_items
```

Comment:
The code just prints things, there is no writing to the file that I can see (or `fileinput` is **really* fancy and I don't know about it -- is it always available?).

Reply:
Yeah, I guess it is fancy by that logic. :)

It takes the output and uses it to update the file in place.